### PR TITLE
tests: use assertThrows instead of try-catch

### PR DESCRIPTION
**Pull request by Meir017**
I wasn't sure what to do about this one in `TestPdf.java`
looks like it only runs on non-chromium
```java
  @Test
  @DisabledIf(value="com.microsoft.playwright.TestBase#isChromium", disabledReason="skip")
  void shouldOnlyHavePdfInChromium() {
    try {
      page.pdf();
      if (isChromium()) {
        return;
      }
      fail("did not throw");
    } catch (PlaywrightException e) {
      assertFalse(e.getMessage().contains("did not throw"));
    }
  }
```

### CODE REVIEW

**Review by yury-s — APPROVED** [review]
Nice! Thanks for doing this!

**Comment by yury-s** [comment]
> I wasn't sure what to do about this one in `TestPdf.java`
> looks like it only runs on non-chromium

Yes, it only runs on Chromium so we should just remove isChromium check and do assertThrows. The test can be renamed to shouldThrowInNonChromium or something like that.

**Comment by Meir017** [comment]
@yury-s Done

File: playwright/src/test/java/com/microsoft/playwright/TestPdf.java [hunk 1]
```diff
@@ -50,16 +50,8 @@ void shouldSupportFractionalScaleValue(@TempDir Path tempDir) throws IOException
...
-      if (isChromium()) {
-        return;
-      }
-      fail("did not throw");
-    } catch (PlaywrightException e) {
-      assertFalse(e.getMessage().contains("did not throw"));
-    }
+  void shouldThrowInNonChromium() {
+    PlaywrightException e = assertThrows(PlaywrightException.class, () -> page.pdf());
+    assertFalse(e.getMessage().contains("did not throw"));
```
Comment (yury-s) [inline playwright/src/test/java/com/microsoft/playwright/TestPdf.java:55]: this is not necessary any more, let's replace it with a check for the actual message.

**Review by yury-s — APPROVED** [review]

**Comment by yury-s** [comment]
I can merge it once the compilation error is fixed.

File: playwright/src/test/java/com/microsoft/playwright/TestPdf.java (hunk 1 above)
Comment (Meir017) [reply playwright/src/test/java/com/microsoft/playwright/TestPdf.java]: Done

**Comment by Meir017** [comment]
pipeline run https://github.com/Meir017/playwright-java/actions/runs/2940207866

**Review by yury-s — APPROVED** [review]

**Pull request merged**